_pages/dev/backend_communication/loading-scopes.md

```diff
@@ -187,3 +187,34 @@ The `reloadOn` configuration allows you to reload a product when a specific even
 ```
 
 In this example, the `details` scope is reloaded when `MyEvent` is emitted.
+
+## Support Scopes for ProductSearch Endpoint
+
+`ProductSearch` endpoint can also be configured to use scopes with limit function now. Scopes in `ProductSearch` don't support `Merging` and `Inclusion` features yet which should be considered when introduce new scope for `ProductSearch`.
```

Comment:
In the first sentence above, is it possible to clarify what is meant by "with limit function"?
Does it refer to `ProductSearch` scopes not supporting `Merging` and `Inclusion`?
Or does it refer to now being able to limit the payload of the ProductSearch endpoint?
Or something else? 🙂

Reply:
"with limit function" means ProductSearch scopes not supporting `Merging` and `Inclusion`